Gemfile

```diff
@@ -8,6 +8,8 @@ gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 4.1'
 gem 'dotenv-rails', '~> 2.7', '>= 2.7.5'
 gem 'dotenv', '~> 2.1', '>= 2.1.1'
+gem 'knock', '~> 2.1', '>= 2.1.1'
```

Comment:
!!!
GET THIS OFF MY GEMFILE!

Reply:
Sir, yes sir ✋😨

Reply:
Sir, fixed in this commit 3d6d14e

Reply:
Oh I see how nice. I'll see to it that I approve! :)